src/schema/extension.schema.ts

```diff
@@ -14,36 +14,10 @@ export interface CXPExtensionManifest {
     title?: string
     description?: string
     readme?: string
-    platform: BundleTarget | DockerTarget | WebSocketTarget | TcpTarget | ExecTarget
+    url: string
     activationEvents: string[]
     args?: {
         [k: string]: any
     }
     contributes?: Contributions & { configuration?: { [key: string]: any } }
 }
-
-export interface BundleTarget {
-    type: 'bundle'
-    contentType?: string
-    url: string
-}
-
-export interface DockerTarget {
-    type: 'docker'
-    image: string
-}
-
-export interface WebSocketTarget {
-    type: 'websocket'
-    url: string
-}
-
-export interface TcpTarget {
-    type: 'tcp'
-    address: string
-}
```

Comment:
We'll need to build and deploy the `.api/xlang` CXP extension before merging this into the browser extension.